cardano-node/cardano-node.cabal

```diff
@@ -76,14 +78,40 @@ library
                        Cardano.Node.Queries
                        Cardano.Node.Run
                        Cardano.Node.STM
+                       Cardano.Node.Startup
+                       Cardano.Node.TraceConstraints
+                       Cardano.Node.Tracing
                        Cardano.Node.Types
+                       Cardano.TraceDispatcher.API
```

Comment:
If this is going to live in cardano-node, can we change the namespace to `Cardano.Node.TraceDispatcher`?

Reply:
@Jimbo4350, done.